### ISSUE TITLE
Running without Express

### ISSUE THREAD

**Issue by yroc92** [thingsup/greenlock-sql-manager#1]
Can this library be used without running the express server? In other words, say I only wanted to get and manage certificates in a SQL database on an ongoing basis but have no desire to open TLS sockets with those certs.

**Comment by Anasnew99** [comment]
No, it does not support running a plain HTTP Server. Since this library is an extension of [greenlock-express](https://www.npmjs.com/package/greenlock-express). 
Although It would be good to remove this limitation. We will look to implement this in feature updates.

**Comment by yroc92** [comment]
Good to know! Thank you.
